### PR TITLE
Fixes test

### PR DESCRIPTION
**Pull request by yuri-zubov**
@abrom I have just fixed tests. I'm looking forward to your comments :)

### CODE REVIEW

File: lib/carmen/rails/action_view/form_helper.rb [hunk 1]
```diff
@@ -175,6 +175,7 @@ def determine_parent(parent_region_or_code)
         when String
           Carmen::Country.coded(parent_region_or_code)
         when Array
+
```
Comment (abrom) [inline lib/carmen/rails/action_view/form_helper.rb]: not sure this newline adds much?!

File: test/carmen/action_view/helpers/carmen_view_helper_test.rb [hunk 2]
```diff
@@ -27,6 +33,10 @@ def test_basic_country_select
     assert_equal_markup(expected, html)
   end
 
+  def document_root_element
+    Nokogiri.parse(@html)
```
Comment (abrom) [inline test/carmen/action_view/helpers/carmen_view_helper_test.rb:37]: what's this do? Can't see any uses

**Review by abrom — COMMENTED** [review]
Nice one @yuri-zubov 👍 

A few questions for you

File: test/carmen/action_view/helpers/carmen_view_helper_test.rb (hunk 2 above)
Comment (yuri-zubov) [reply test/carmen/action_view/helpers/carmen_view_helper_test.rb:37]: function `assert_select` wants to see `document_root_element` I don't know how to make document_root_element exactly.
 I made function `document_root_element` and parsed variable @html that existing in each methods :(. If you have an alternative hack, please tell me

File: lib/carmen/rails/action_view/form_helper.rb (hunk 1 above)
Comment (yuri-zubov) [reply lib/carmen/rails/action_view/form_helper.rb]: it was a place for `binding.pry`

File: test/carmen/action_view/helpers/carmen_view_helper_test.rb (hunk 2 above)
Comment (abrom) [reply test/carmen/action_view/helpers/carmen_view_helper_test.rb:37]: Fair enough :+1:

**Comment by yuri-zubov** [comment]
@abrom connect your repository to TravisCI(example https://travis-ci.org/yuri-zubov/carmen-rails)
and you will see checking points in your commits
image

**Pull request merged**